android/java/res/layout/brave_sync_add_laptop.xml

```diff
@@ -52,48 +31,86 @@
             android:paddingEnd="15dip"
             android:paddingBottom="15dip">
 
-            <TextView android:id="@+id/brave_sync_text_add_laptop"
+            <TextView
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:gravity="center" />
+                android:text="@string/brave_sync_add_laptop_text_title"
+                style="@style/BraveSyncTitleText"
+                android:gravity="top" />
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="@string/brave_sync_add_laptop_text"
+                android:gravity="top"
+                android:layout_marginTop="20dp"
+                style="@style/BraveSyncDescriptionText"  />
 
             <TextView android:id="@+id/brave_sync_warning_text_add_laptop"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:gravity="center" />
+                style="@style/BraveSyncRedWarningText"
+                android:text="@string/brave_sync_code_warning"
+                android:layout_marginTop="20dp" />
 
         </LinearLayout>
 
-        <FrameLayout android:layout_width="match_parent"
-            android:layout_height="0px"
-            android:layout_weight="1"
+        <FrameLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
             android:paddingStart="15dip"
-            android:paddingEnd="15dip" >
+            android:paddingEnd="15dip"
```

Comment:
nit :  you can use android:paddingHorizontal="15dp"

Reply:
thanks fixed at 309e389c8db4ebc9f653667a344389dfb089b834